src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        if (userService.findByEmail(email) == null) {
+            return false;
+        }
+        return userService.findByEmail(email).getPassword().equals(password);
```

Comment:
I would recommend to fetch user at the beginning, just to avoid calling `findByEmail` each time any check is conducted